integration_test/third_party_apps_data/applications/couchbase/metadata.yaml

```
@@ -34,31 +38,51 @@ expected_logs:
     description: ''
 - log_name: couchbase_http_access
   fields:
-  - name: jsonPayload.message
+  - name: httpRequest.remoteIp
+    value_regex: ::1|127.0.0.1
     type: string
-    description: "Log message"
-  - name: jsonPayload.client_ip
+    description: The IP address (IPv4 or IPv6) of the client that issued the HTTP request
+    optional: true
+  - name: httpRequest.requestUrl
     type: string
-    description: The IP address of the client invoking the HTTP request
+    description: Request URL (typically just the path part of the URL)
     optional: true
-  - name: jsonPayload.user
+  - name: httpRequest.protocol
     type: string
-    description: The name of the user making the HTTP request if basic auth is used.
+    description: Protocol used for the request
     optional: true
-  - name: jsonPayload.method
+  - name: httpRequest.requestMethod
+    value_regex: GET
     type: string
-    description: The HTTP method of the request
-  - name: jsonPayload.response_size
+    description: HTTP method
+    optional: true
+  - name: httpRequest.status
+    type: number
+    description: HTTP status code
+    optional: true
+  - name: httpRequest.responseSize
     type: string
-    description: The HTTP response size
+    description: ''
     optional: true
-  - name: jsonPayload.path
+  - name: httpRequest.userAgent
     type: string
     description: ''
     optional: true
-  - name: jsonPayload.status_code
-    type: integer
-    description: The status code of the response of the HTTP request
+  - name: jsonPayload.message
+    type: string
+    description: "Log message"
+  - name: jsonPayload.host
+    type: string
+    description: The IP address of the client invoking the HTTP request
+    optional: true
+  - name: jsonPayload.level
+    type: string
+    description: Log entry severity for the couchbase log
+    optional: true
+  - name: jsonPayload.user
+    type: string
+    description: The name of the user making the HTTP request if basic auth is used.
+    optional: true
```

```suggestion
  - name: jsonPayload.level
    type: string
    description: Log entry severity for the couchbase log
    optional: true
```

I'd expect these to be in the httpRequest field right?

`jsonPayload.host => httpRequest.remoteIP`
`jsonPayload.user => httpRequest.userAgent`

---

The host != remoteIp and user != userAgent as part of https://github.com/GoogleCloudPlatform/ops-agent/blob/d8367b79d3154a0bac8831cc0d38dea4f686c4e4/apps/common_logging_processors.go#L33-L64

---

Ahh I see now,  thanks for clarifying!